ProjectHive/ProjectHive.Service.Core/Data/Repository/Repository.cs

```diff
@@ -77,4 +77,20 @@ public async Task Commit()
     {
         await _dbContext.SaveChangesAsync();
     }
+
+    public IQueryable<TEntity> FindBy(Expression<Func<TEntity, bool>> expression, params Expression<Func<TEntity, object>>[] includes)
+    {
+        var resultQuery = _dbSet.Where(expression);
+        if (includes.Any())
+        {
+            resultQuery = includes.Aggregate(resultQuery,
+                (current, include) => current.Include(include));
+        }
+        return resultQuery;
+    }
+
+    public async Task<List<TEntity>> FindBy()
+    {
+        return await _dbSet.ToListAsync();
+    }
```

Comment:
Please \, do not forget about cancellation tokens, add them to all async methods there, `_dbSet.ToListAsync(cancellation Token);` and so on

Reply:
Also, fix the namespace, it does not correspond the current location